python/py-meson/Portfile

```diff
@@ -71,6 +71,15 @@ if {${subport} ne ${name}} {
 
     # add a search path for crossfiles in our prefix
     patchfiles-append   patch-meson-search-prefix-for-cross-files.diff
+    
+    # temporary fix for the destroot failure on a lot of packages using meson build
+    # Since version release 1.5.0 on darwin platform.
+    # This fix is poroposed to mesonbuild by :
+    # holymonson (Monson Shao)
+    # This patch will need to be removed by next meson release upgrade if 
+    # meson solved the problem upstream on next release.
```

Comment:
Instead of these comments please just add a link to the Trac ticket instead. 

Also in your commit message, please just say "Closes: <full URL to Trac ticket" to automatically close it. No need for any if the other text.